FunnyPuny/Presentation/Home/HomeViewController.swift

```diff
@@ -6,14 +6,22 @@ import RealmSwift
 import SwiftDate
 import UIKit
 
-// swiftlint:disable all
+// MARK: - HomeViewController
+
+enum HomeState {
```

Comment:
enum need to be inner in the class + private

Reply:
enum needs to be public, because it is used in home view model